Source/WebCore/dom/Subscriber.cpp

```diff
@@ -133,17 +132,13 @@ void Subscriber::close(JSC::JSValue reason)
 
 bool Subscriber::isInactiveDocument() const
 {
-    RefPtr document = dynamicDowncast<Document>(scriptExecutionContext());
+    RefPtr document = dynamicDowncast<Document>(protectedScriptExecutionContext());
```

Comment:
Please revert this change, there is no benefit to calling `protectedScriptExecutionContext()` here and it makes the code less concise for no good reason.